torch/_meta_registrations.py

```diff
@@ -80,6 +80,9 @@ def meta_randperm(n, *, generator=None, out):
     assert out.ndim == 1 and out.size(0) == n
     return out
 
+@register_meta(aten.randint.default)
+def meta_randperm(high, size, dtype, layout, device, pin_memory):
+    return torch.empty(size, dtype=dtype, device=device)
```

Comment:
Why not pass layout and pin memory?

Reply:
oh yeah that needs to be updated

Reply:
one idiom for factory functions is to just pass along the entire `**kwargs`